shell/common/switches.cc

```diff
@@ -78,13 +80,13 @@ void PrintUsage(const std::string& executable_name) {
   std::cerr << std::string(column_width, '-') << std::endl;
 }
 
-const char* FlagForSwitch(Switch swtch) {
+const ftl::StringView FlagForSwitch(Switch swtch) {
```

Comment:
Yes, that was my suggestion. The FTL update is the point, it allows all this code to build successfully on Fuchsia.